packages/spec/integration/postcss/index.js

```diff
@@ -1,11 +1,21 @@
 import { render } from 'hops';
 import React from 'react';
+import { Helmet } from 'react-helmet-async';
 import styles from './styles.css';
 /* eslint-disable import/no-unresolved */
 import './global.css?global';
 import 'animate.css/animate.min.css?global';
 /* eslint-enable import/no-unresolved */
 
 export default render(
-  <h1 className={`${styles.headline} fancy-headline animated`}>hello</h1>
+  <>
+    <Helmet>
```

Comment:
Do we still need `Helmet` here? Because the favicon issue only appears when enabling debug mode and it is not directly related to the changes in this PR.

Reply:
I'll move that to another PR